modules/time.js

```diff
@@ -0,0 +1,17 @@
+// timer
+const displayDate = () => {
+  const date = new Date();
+  const options = {
+    weekday: undefined,
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  };
+  const [month, time] = [
+    date.toLocaleDateString(undefined, options),
+    date.toLocaleTimeString().toLocaleLowerCase(),
+  ];
+  document.querySelector('.time').innerHTML = `${month}, ${time}`;
+};
+
+export default displayDate;
```

Comment:
- [ ] It is required from the project requirements to use [luxon](https://moment.github.io/luxon/#/install?id=es6) to format your date. Kindly abide by this specification. Take a look from the project requirements.

![your-date](https://user-images.githubusercontent.com/26597432/199119338-56fc2186-d8c9-47d2-9cda-62c0f5645597.png)